build/ci.go

```diff
@@ -0,0 +1,61 @@
+// +build none
```

Comment:
can we move this to `scripts/`? not a fan of having a non-disposible `build` directory

Reply:
sure